Please review and comment

zachdimitrov opened the pull request:
- New buildings added
- Changed field dimentsions to open space for sidebar
- Sidebar created (removed all weird symbols that slowed down the game)
- Moved scoreboard to sidebar but not cleared Message Box code from Engine
  - не се с правих с това как мога да ползвам Message Box за да създам страничната лента, може bи трябва да се преправи за да няма толкова много Console.WriteLine, не съм трил нищо от кода в Engine, само закоментирах ред 262 ...

zachdimitrov committed:
Created side menu panel

zachdimitrov committed:
Created new buildings and modified ShootingTower

zachdimitrov committed:
Merge branch 'master' of https://github.com/Team-Hestia/ThunderFighter

# Conflicts:
#	ThunderFighter/Engine.cs

zachdimitrov committed:
Corrected sidebar and moved scoreboard

madmarks commented:
Avoid using Console.Write and Console.WriteLine. Instead of these Console class methods try to use ScreenBuffer.Draw method. Look at class Pixel how to use it.

zachdimitrov committed:
- created new Collision Detections
 - bullet - bullet
 - player - building
 - bomb - enemy
 - player - bullet
- corrected sidebar draw with ScreenBuffer
- filled sidebar with info (not updating)

zachdimitrov commented:
- Corrected sidebar drawing with ScreenBuffer.Draw()
- Rearanged Field - added PlayWidth/Height and MenuWidth/Height
- Created more collision detections (mostly copy - paste)
- Added enumeration for different scores depending on what is killed
- Implemented calculation of score
- Implemented levels, depending on score
- Filled Menu info - but live-update of all info is not working?!
  - опитах се по всякакъв начин но не успях да накарам инфо-то да се променя в движение, някъде греша, ако може да помогнете къде? Като натисна P се появяват натрупаните точки и т.н., но не и по време на игра. Не разбирам и защо брояча и таймера работят по странен начин, уж правя както трябва, явно и тук греша, помагайте :)

zachdimitrov committed:
Implemented themes, change theme color from Field instance in Game class (ThemeColor)

zachdimitrov commented:
- Color themes implemented - white, black, blue, see latest commit
